config/config.exs

```diff
@@ -33,6 +33,14 @@ config :phoenix, :generators,
   migration: true,
   binary_id: false
 
+config :mime, :types, %{
+  "*/*" => ["json"],
+  "application/json" => ["json"],
+  "application/binary" => ["binary"],
+  "application/avro.binary" => ["avro"],
```

Comment:
you mean, why do we need to handle binary and avro.binary content types in client requests?